src/App.css

```diff
@@ -0,0 +1,55 @@
+/* blue */
+* {
+  box-sizing: border-box;
+  --main-color-very-dark: #1a365d;
+  --main-color-dark: #2c5282;
+  --main-color-medium: #3182ce;
+  --main-color-light: #63b3ed;
+  --main-color-very-light: #bee3f8;
+  --grey-dark: #4a5568;
+  --grey-medium: #a0aec0;
+  --grey-light: #e2e8f0;
+  --white: #fff;
```

Comment:
This is pointless as `white` as a css color keyword is already this hexcode.  If you wanted to use a slightly offwhite (something many designers encourage) then this would make sense

Reply:
Yeah, I figured this way if I want to play around with off-white colors it's all set up.